src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -2,20 +2,41 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 public class DatabaseManager {
     private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
 
     private Connection connection;
 
-    boolean connect(String database, String user, String password) {
+    public boolean connect(String database, String user, String password) {
         try {
             Class.forName("org.postgresql.Driver");
-            connection = DriverManager.getConnection(DB_CONNECTION_URL + database, user, password);
+            connection = DriverManager.getConnection(DB_CONNECTION_URL + database+"?loggerLevel=OFF", user, password);
             return true;
         } catch (ClassNotFoundException | SQLException e) {
             return false;
         }
     }
+
+    public String[] getTableNames() throws SQLException {
+        String sqlQuery = "SELECT relname FROM pg_stat_user_tables";
```

Comment:
How did you find this query? If I google such string there are no results. Anyway despite simplicity of the query, I think it's better to use Metadata, because it's more db independent

Reply:
I find this query when was making a code review for Alexander's (avm) code, so it's better to ask him))) And I also told him that this query  is dependent on DB, but I find this query pretty interesting an decide to use it... But you are totally wright that using DatabaseMetaData is db independent. Just for now I'd like to leave it like it is.
Thanks for your remark!

Reply:
Cool